AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -644,11 +644,24 @@ - (void)startWithChannelGroup:(id<MSChannelGroupProtocol>)channelGroup
     self.httpClient = [MSHttpClient new];
   }
 
-  // Listen to network events.
-  [MS_NOTIFICATION_CENTER addObserver:self selector:@selector(networkStateChanged:) name:kMSReachabilityChangedNotification object:nil];
+  // Listen to notificaitons.
```

Comment:
```suggestion
  // Listen to notifications.
```

Reply:
Although you shouldn't have to do it at start time. The service abstraction will actually call the `applyEnabledState` if the SDK is enabled while starting.